src/main/webapp/src/app/hidden-gem.service.spec.ts

```diff
@@ -153,7 +132,7 @@ describe('HiddenGemService', () => {
 
     httpClientSpy.get.and.returnValue(of(errorResponse));
 
-    hiddenGemService.findHiddenGemRecommendation("").subscribe(
+    hiddenGemService.getRankedHiddenGems(LOCATION).subscribe(
```

Comment:
this error test initially for recommendation can also be removed i think. Its a duplicate test